python/tk_multi_publish2/publish_tree_widget/publish_tree_widget.py

```diff
@@ -115,8 +115,12 @@ def _build_item_tree_r(self, item, enabled, level, tree_parent):
 
         # create children
         for task in item.tasks:
-            task = TreeNodeTask(task, ui_item)
-            self.__created_items.append(task)
+            ui_task = TreeNodeTask(task, ui_item)
```

Comment:
I changed this because the iteration variable `task` was being set which was confusing. Also the "task" item vs. the actual "task" plugin. I'll look to clarify this when rewiring for publish API.